src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classes from './ErrorMessage.module.scss';
+import { ExclamationCircleIcon } from '@heroicons/react/24/outline';
+
+interface ErrorMessageProps {
+  message: string;
+}
+
+const ErrorMessage: React.FC<ErrorMessageProps> = ({ message }) => {
+  return (
+    <div className={`${classes['c-error-message']}`}>
+      {message && (
```

Comment:
Also message is not optional so no need for conditional rendering or at least with `&&` it will not check for empty string